ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/JdbcLedgerDao.scala

```diff
@@ -114,6 +114,11 @@ private class JdbcLedgerDao(
   override def lookupLedgerEnd()(implicit loggingContext: LoggingContext): Future[Offset] =
     dbDispatcher.executeSql(metrics.daml.index.db.getLedgerEnd)(ParametersTable.getLedgerEnd)
 
+  // TODO
+  override def lookupLedgerEndAndEventSequentialId()(implicit
+      loggingContext: LoggingContext
+  ): Future[(Offset, Long)] = ???
+
```

Comment:
I wasn't sure on the strategy for the mutable state cache vs parallel ingestion development.
@nmarton-da @tudor-da Which package is appropriate for the mutable state cache work - `dao` or `appendonlydao`?